ktor-server/ktor-server-host-common/jvm/src/io/ktor/server/engine/BaseApplicationResponse.kt

```diff
@@ -239,21 +242,39 @@ abstract class BaseApplicationResponse(override val call: ApplicationCall) : App
      * [OutgoingContent] is trying to set some header that is not allowed for this content type.
      * For example, only upgrade content can set `Upgrade` header.
      */
-    class InvalidHeaderForContent(name: String, content: String) : IllegalStateException("Header $name is not allowed for $content")
+    class InvalidHeaderForContent(
+        private val name: String, private val content: String
```

Comment:
Looks like this class is public, does this change break compatibility (I mean `public` -> `private` replacement)?

Reply:
There were no extra fields in this class before, only constructor params